CHANGELOG

```diff
@@ -1,40 +1,126 @@
-0.3.0dev0
-=========
-- Added support for Ipopt >=3.13 on Windows.
-  (https://github.com/matthias-k/cyipopt/pull/63)
-
-0.2.0
-=====
-
-- Resolved compatibility issues with Windows.
-  (https://github.com/matthias-k/cyipopt/pull/49)
-- OSError now raised if pkg-config can't find ipopt on installation.
-  (https://github.com/matthias-k/cyipopt/pull/57)
-- Supporting only Python 2.7 and 3.6-3.8. Python 3.5 support dropped.
-  (https://github.com/matthias-k/cyipopt/pull/58)
-- Improvements to the README and setup.py for Windows installations.
-  (https://github.com/matthias-k/cyipopt/pull/54)
-- Drop Python 3.4 support and add Python 3.7 support.
-  (https://github.com/matthias-k/cyipopt/pull/51)
-- Adding installation testing on the Appveyor CI service.
-  (https://github.com/matthias-k/cyipopt/pull/50)
+CyIpopt Changelog
+==================
+
+:Info: Change log for CyIpopt releases.
+:Authors: Matthias Kuemmerer (matthias.kuemmerer@bethgelab.org), Jason Moore (moorepants@gmail.com), Sam Brockie (sgb39@cam.ac.uk)
```

Comment:
I created an AUTHORS file, so maybe remove this line. We can track AUTHORS strictly in that file so we don't have to update multiple places.

Reply:
Okay, will look at that, thanks.

Reply:
Addressed in https://github.com/matthias-k/cyipopt/pull/75/commits/10285489111ee5d5d82521ec0d1172cb9e45f69f and https://github.com/matthias-k/cyipopt/pull/75/commits/c74c9a26884af623cfeb8ef4557b586a78391d46.